app/controllers/billing.js

```diff
@@ -1,6 +0,0 @@
-import Controller from '@ember/controller';
-import {alias} from '@ember/object/computed';
-
-export default Controller.extend({
-    guid: alias('model')
-});
```

Comment:
I'd recommend keeping this file because a route always has a controller (it's created automatically for you in the bg if no explicit file exists) and it's useful to set the initial state for any any query params. Query params are always stored on the controller so an `action: null` property is useful purely for explicitness and to know what it's empty/base state is.

Reply:
Oh cool, this makes a lot of sense. Thanks!

Reply:
Oh, the query params will also need to be defined here with `queryParams: ['action']`. The object in the route file is only used to adjust the refresh behaviour.